clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsRequest.java

```diff
@@ -31,7 +31,8 @@
 public class ApiVersionsRequest extends AbstractRequest {
 
     public static class Builder extends AbstractRequest.Builder<ApiVersionsRequest> {
-        private static final String DEFAULT_CLIENT_SOFTWARE_NAME = "apache-kafka-java";
+        private static final String DEFAULT_CLIENT_SOFTWARE_NAME = "linkedin-kafka-oss-java";
+        private static String customizedClientName = null;
 
         private static final ApiVersionsRequestData DATA = new ApiVersionsRequestData()
```

Comment:
should we make DATA non-static and use it directly instead of deferring the logic to build function. 

e.g., 
builder (String clientName) {
DATA = new ApiVersionsRequestData()
            .setClientSoftwareName(clientName)
            .setClientSoftwareVersion(AppInfoParser.getVersion());
}